docs/edgeql/statements/describe.rst

```diff
@@ -0,0 +1,194 @@
+.. _ref_eql_statements_describe:
+
+DESCRIBE
+========
+
+:eql-statement:
+
+``DESCRIBE`` -- provide human-readable description of a concept
+
+.. eql:synopsis::
+
+    DESCRIBE SCHEMA [ AS {DDL | SDL | TEXT} ] [ VERBOSE ];
+
+    DESCRIBE <concept-type> <name> [ AS {DDL | SDL | TEXT} ] [ VERBOSE ];
+
+    # where <concept-type> is one of
+
+      OBJECT
+      ANNOTATION
+      CONSTRAINT
+      FUNCTION
+      LINK
+      MODULE
+      PROPERTY
+      SCALAR TYPE
+      TYPE
+
+Description
+-----------
+
+``DESCRIBE`` generate a human-readable description of a schema object.
+
+The output of a ``DESCRIBE`` command is a :eql:type:`str` , although
+it cannot be used as an expression in queries. It is intended as a
```

Comment:
The language is confusing.  Are you referring to the command or it's output?  I would say that `DESCRIBE` is a command and not an expression.

Reply:
Well, it looks like a `str` in REPL. It returns a `str` when I fetch it, I think. Or am I mistaken? It's illegal to use it as an expression, but the command still returns something and that something has the formatting of a `str`.